src/components/DigitalCredentials/CredentialsWebSocket.vue

```diff
@@ -3,39 +3,95 @@
 </template>
 
 <script lang="ts">
-import { Component, Emit, Vue } from 'vue-property-decorator'
-import { io } from 'socket.io-client'
-import { useConfigurationStore } from '@/stores'
+import { Component, Emit, Prop, Vue, Watch } from 'vue-property-decorator'
+import { useBusinessStore, useConfigurationStore } from '@/stores'
 import { Getter } from 'pinia-class'
-import { WebSocketEvents, WebSocketTopics } from '@/enums'
 import { WalletConnectionIF, DigitalCredentialIF } from '@/interfaces'
+import { Observable, Subscription, interval, of } from 'rxjs'
```

Comment:
I've used observables (and rxjs) before and _I like it_ :)

Reply:
Felt easier than writing a whole implementation in JS. Why re-invent the wheel right?